src/GeneralStateTestsFiller/stCreateTest/CreateOOGafterInitCodeReturndata3Filler.json

```diff
@@ -0,0 +1,75 @@
+{
+    "CreateOOGafterInitCodeReturndata3" : {
+	"_info" : {
+	    "comment" : "Calls a contract that runs CREATE which deploy a code. then OOG happens upon deployment of the actual code. check the RETURN data buffer in initial contract"
+	},
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "10000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+	    {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : [">=Byzantium"],
+                "result" : {
+		    "c94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+		        "storage" : {
+                                "0x01" : "0x00"
+                         }
+		    },
+		    "0xf1ecf98489fa9ed60a664fc4998db699cfa39d40" : {
+		        "shouldnotexist" : "1"
+		    }
+                }
+            }
+        ],
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "c94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0",
+		"nonce" : "0",
+		"code" : "{ (CALLCODE (GAS) 0xb94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 32) (RETURNDATACOPY 0 0 32) [[ 1 ]] (MLOAD 0) }",
+		"storage": {
+		}
+            },
+	    "b94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0",
+		"nonce" : "0",
+		"//code" : "(SSTORE 1 1) RETURN",
+		"code" : "{ (MSTORE 0 0x6460016001556000526005601bf3) (CREATE 0 18 14) }",
```

Comment:
You were supposed to check returndata after the `CREATE`, not to the higher level of `CALLCODE`.

Reply:
Right. It is just that if I put any more instructions father it wont OOG anymore.